Getters/pricingPage.json

```diff
@@ -0,0 +1,20 @@
+{
+	"acceptCookies" : "a[id='cookieGotIt']",
+	"monthlySwitch" : "div[class='vsp-c-switch-pricing-plan-option'] > p:contains('Monthly')",
```

Comment:
Nice idea to use `contains`, however, what if we wanted to test this in e.g. German? It would then become `Monatlich` and the test would fail. We need a less brittle selector, especially if we ever plan to test i18n the app.